_data/sites.json

```diff
@@ -5209,6 +5209,17 @@
             "sedo.com"
         ]
     },
+    
```

Comment:
This trailing space will cause CI to fail--please remove the extra spaces on this line.